client/coral-embed-stream/src/containers/Stream.js

```diff
@@ -113,6 +113,7 @@ class StreamContainer extends React.Component {
   componentDidMount() {
     if (this.props.previousTab) {
       this.props.data.refetch();
+      this.props.setCommentCountCache(0);
```

Comment:
I think `refetch` is a promise and you can set the commentCount to the correct value when the promise returns the new result. The whole `commentCount` thing will be gone as soon as we have an alternative subscription based implementation.